documentation/final-deliverables/pdf-report.md

```diff
@@ -2,110 +2,109 @@
 
-FR19. The Admin will decide how to host the Mentorship form (either publicly, or internally through email invite). The source code for publicly embedded forms will be generated by clicking a "Generate" button so that Admins can add the iFrame tag to their organization's public website.
+FR14. Create a static Mentor form with name, email, time zone, calendar link, password, confirm password, top three skills, and description fields.
 
-FR20. The form will be dynamic in the way that Admins can edit the fields of the form to suit their needs.
+FR15. Associate the Mentee and Mentor forms with the Admin's Organization that the form was accessed through so Mentees and Mentors only match from the same organization.
 
-FR21. The form will be dynamic in the way that depending on the user filling out the form on public sites (i.e either mentor or mentee), once the radio button is selected for the type of user, the form will display appropriate information for each.
+FR16. Account registration for Mentors requires a unique email address, and a password. To register an account successfully, the user must be invited by an admin.
 
-FR22. The email that notifies Mentees that they have been matched to a Mentor will link to a third-party calendar (such as Google Calendar) with the Mentor's availability. Mentees will be able to create an appointment with Mentors through that calendar.
+FR17. A verification email will be sent to the address provided in account registration to Mentors and Admins. The user must click that link to be able to login to their account so that they can access the features specific to their user type.
 
-FR23. To "unmatch" Mentees and Mentors, the Mentee can remove themselves from the calendar, cancelling all future appointments. The Mentor could then remove them for their dashboard on the Mentorship Portal.
+FR18. Mentors and Admins will be able to login through MentorMatch's login page. The login will be unsuccessful if either the email or password credentials do not align with what’s in the database (in which case, an error message will be displayed).
 
-FR24. Mentors who validate their email address will be able to login, however, they will not be able to view any matched Mentees until an Admin approves their account.
+FR19. Mentors and Admins will be able to reset their password by clicking on the “Forgot your password?” link on the Login page. The user will be required to enter an email address which has been associated with an account in the database. If successful, an email will be sent to that address, which the user can click to be brought to a separate page to reset their password.
 
-FR25. Admins will be able to approve Mentors through the Mentorship Portal.
+FR20. Mentors and Admins will be able to logout of their account, to which they will return to the login page, where they can login, retrieve their password, or register.
```

Comment:
```suggestion
FR20. Mentors and Admins will be able to logout of their account, taking them to the login page, where they can login, retrieve their forgotten password, or register.
```

Reply:
My bad, poor wording on my end. I meant that by going to the Google Calendar, an individual can click on an event on the calendar, and provide a message stating that they'd like to be mentored at that time (I've attached a screenshot of this below to help visualize what I'm referring to). I can definitely see how this wasn't reflected in how I originally phrased things! Mentors having to manually update their calendars to reflect booked time
slots is definitely a limitation, but [this](https://support.google.com/calendar/answer/190998?co=GENIE.Platform%3DDesktop&hl=en) provides a simple fix that we can note in our documentation later.

<img width="930" alt="Screen Shot 2021-03-23 at 3 39 11 PM" src="https://user-images.githubusercontent.com/20407302/112228080-ed51c080-8bed-11eb-9f06-141948c6f614.png">
<img width="659" alt="Screen Shot 2021-03-23 at 3 40 16 PM" src="https://user-images.githubusercontent.com/20407302/112228158-140ff700-8bee-11eb-8cca-4aab2b25b8fe.png">


For simplicity, would the following modification be alright?
```suggestion
  By checking my emails, I'm eventually notified that a Mentor has accepted my request. Within this email, I can see the Mentor's name, their email address, a description about the individual, and a link to their calendar. Navigating to their calendar allows me to view the Mentor's availability so that I can request a time to be mentored.
```